File: tests/samples_test.py
#!/usr/bin/env python
from tests.parser_test_helper import *

class SamplesTest(ParserBaseTest):

    def setUp(self):
      super(SamplesTest, self).setUp()

      # context._verbose = True
      self.parser.clear()

    def test_addition(self):
      x=parse("samples/addition.e")
      assert x, "parsed"
      assert_result_is("add 7 to 3","10")

    def test_hello_world(self):
      x=parse("samples/hello-world.e")
      assert_equals(x,"hello world")

File: angle.js/test/samples.test.js
#!/usr/bin/env python;


class SamplesTest extends (ParserBaseTest) {
	setUp() {
		// super(SamplesTest, self).setUp();
		parser.clear();
	}

	// context._verbose = True

	test_addition() {
		x = parse(`samples/addition.e`);
		assert(x, 'parsed')
		assert_result_is(`add 7 to 3`, '10');
	}

	test_hello_world() {
		x = parse(`samples/hello-world.e`);
		assert_equals(x, 'hello world');
	}
}
register(SamplesTest, module)
